languages/rust/exercises/concept/string-vs-str/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+pub struct CSVRecordBuilder {
+    content: String
+}
+
+impl CSVRecordBuilder {
+    // Create a new builder
+    pub fn new() -> Self {
+        unimplemented!()
```

Comment:
To help the student figure out what to do, we often pass in an argument to let the student know what to implement:

```suggestion
        unimplemented!("Please implement the `CSVRecordBuilder.new` function")
```

or something like that. See [this Rust exercise](https://github.com/exercism/v3/blob/master/languages/rust/exercises/concept/tuples/src/lib.rs#L2) for something similar.